release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2068,6 +2068,8 @@ In various circumstances, the management state of the Cluster Samples Operator c
 
 * Before this update, the tooltip for tasks in the *PipelineRun details* showed misleading information. It showed the time elapsed since the task ran, not how long they ran. For example, it showed 122 hours for a task that ran for a couple of seconds 5 days ago. With this update, the tooltip shows the duration of the task. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011368[BZ#2011368])
 
+* Before this update, when you use the *Import from Git* form to import a private git repository, the correct import type and the builder image are not identified. This issue used to happen because the secret to fetch the private repository details is not decoded. This update resolves the issue by decoding the secret value before using it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```

Comment:
```suggestion
* Before this update, when a secret is provided for a private repository in the *Import from Git* form, the value of the secret to fetch the private repository details is not decoded before being used. As a result, when you use the *Import from Git* form to import a private Git repository, the correct import type, and the builder image are not identified. This update resolves the issue by decoding the secret value before using it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```

Reply:
Accepting the changes. Thanks @Srivaralakshmi